godotproject/project.godot

```diff
@@ -179,6 +179,10 @@ menu={
 [rendering]
 
 quality/driver/driver_name="GLES2"
+quality/2d/use_pixel_snap=true
 vram_compression/import_etc=true
 vram_compression/import_etc2=false
+batching/precision/uv_contract=true
 environment/default_environment="res://default_env.tres"
+batching/options/single_rect_fallback.Pi=true
+batching/parameters/max_join_item_commands.Pi=64
```

Comment:
The "single rect. fallback" and "max join items" will be applied only on Pi, since we don't need them on other platforms and they could introduce issues.